baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -372,14 +372,33 @@ export class AddressesClient {
       'project': request.project || '',
     });
     this.initialize();
-    return this.innerApiCalls.delete(request, options, callback);
+    return new Promise<[
+        LROperation<protos.google.cloud.compute.v1.IOperation, null>,
+        protos.google.cloud.compute.v1.IOperation|undefined, {}|undefined
+      ]>((resolve, reject) => {
+      this.innerApiCalls.delete(request, options, callback)
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      .then(([response, next, rawResponse]: [any, any, any]) => {
+        resolve([
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-expect-error
```

Comment:
I have question to use duck typing for interface [gax.Operation](https://github.com/googleapis/gax-nodejs/blob/acedd552199f57ffae23ed0c41464304fc3bb6f2/src/longRunningCalls/longrunning.ts#L46). The current object has only 3 fields: `latestResponse, done, name`, ts linter compliant that miss other fields in [Operation](https://github.com/googleapis/gax-nodejs/blob/acedd552199f57ffae23ed0c41464304fc3bb6f2/src/longRunningCalls/longrunning.ts#L46).
Option 1(fix the linter):
If I try to pass non-optional value for the rest fields, there are two fields I am not sure how to sign value `longrunningDescriptor` and `backoffSettings`. We need to construct these two objects but eventually will replace after full-implementation.

Option2(ignore linter)
Another option is what I did here, ignore the linter. And save some time for full implementation and remove entire wrapper as soon as we fully implement diregapic lro.

@alexander-fenster, @bcoe can I have some advice here. Or maybe we have easy quick way to fix.

Reply:
You can put a fake LongRunningDescriptor (just create a new one) and a fake backoff settings (again, there should be a way to create a default backoff settings object). Replacing them with real stuff later won't break compatibility.

Reply:
Oh, sure. Thanks for clarification. I will go ahead to return the response same as LROperation.

One reason I hesitate to create a fake LongRunningDescriptor is because it surface the fake object to user in the generated client library. I don't have a good reason say that we should not do this. Just not super sure. Now I got more confidence to do so.